e2e/helpers/language.js

```diff
@@ -1,9 +1,15 @@
+import merge from 'lodash/merge';
+
 import * as english from '../../app/locales/en.json';
 import * as haitian from '../../app/locales/ht.json';
 
+/**
+ * Language strings for each locale. Strings are merged with english as a base
+ * to emulate i18next's english fallback in the case of missing locale strings.
+ */
 const languageStrings = {
   'en-US': english,
-  'ht-HT': haitian,
+  'ht-HT': merge({}, english, haitian),
```

Comment:
How would you feel about moving this `merge` logic down into the map function? e.g.

```js
export const languages = Object.keys(languageStrings).map(locale => [
  locale,
  merge({}, english, languageStrings[locale]),
]);
```

Reply:
Way better, will do